cfgov/jinja2/v1/_includes/organisms/featured-content.html

```diff
@@ -6,94 +6,54 @@
+   link.text:       (Optional) Link text.
 
-   value.post_link_text:  String for the link text of the displayed post.
-
-   value.image:           (Optional) An ImageBasic atom object containing:
-
-   value.image.upload:    A CFGOVImage object.
-
-   value.image.alt:       String for alt text for screenreaders.
-
-   value.links:           If the FCM is rendered in wagtail,
-                          this returns the array of links.
-
-   value.links[i].url:    String for URL of the link.
-
-   value.links[i].text:   String for text of the link.
-
-   value.video:           (Optional)
-                          This object gets passed to macros/video-player.html.
-                          By default, the following values should exist:
-                            'is_fcm': true,
-                            'height': '320',
-                            'width':  '568'
-                          Wagtail should set the following values:
-                            'id', 'url'
-
-   value.video.url:       A String used to denote the url of the video.
-
-   value.video.id:        A String used to denote the id of the video.
+   value.image:     (Optional) An ImageBasic atom.
 
+   value.video:     (Optional) A VideoPlayer organism.
 
    ========================================================================== #}
 
-{% import 'macros/category-icon.html' as category_icon %}
-
 <section class="o-featured-content-module
                 o-featured-content-module__center">
     <div class="o-featured-content-module_text">
         <h2 class="h3">{{ value.heading }}</h2>
-        {{ value.body | safe }}
-        {% if (value.show_post_link and value.post)
-           or (value.links[0] and value.links[0].text and value.links[0].url) %}
-            <ul class="m-list m-list__links">
-                {% if value.show_post_link and value.post_link_text and value.post %}
-                    <li class="m-list_item">
-                        <a href="{{ pageurl(value.post) }}">
-                            {{ value.post_link_text }}
-                        </a>
-                    </li>
-                {% endif %}
-                {% if value.links[0] and value.links[0].text and value.links[0].url %}
-                    {% for link in value.links %}
-                        {% if link.text and link.url %}
-                            <li class="m-list_item">
-                                <a href="{{ link.url }}">
-                                    {{ link.text }}
-                                </a>
-                            </li>
-                        {% endif %}
-                    {% endfor %}
-                {% endif %}
-            </ul>
+
+        {{ value.body | richtext }}
+
+        {% if value.links %}
+        <ul class="m-list m-list__links">
+            {% for link in value.links %}
+                <li class="m-list_item">
+                    <a href="{{ link.url }}">{{ link.text | default( link.url ) }}</a>
```

Comment:
Link list [here in the DS](https://cfpb.github.io/design-system/components/lists#link-list) has the link as an element of `m-list`, with `class="m-list_link"` on the link. Does adding that screw anything up?

Reply:
Nice find. It turns out that this does pretty significantly change the behavior of these links, both on desktop and mobile.

The font weight gets increased to 500, and on mobile these links get a bigger touch target and horizontal lines separating them.

[Page with single link](https://www.consumerfinance.gov/consumer-tools/auto-loans/), desktop and mobile:

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/654645/84791458-9fee5600-afc0-11ea-9581-d81715367672.png)|![image](https://user-images.githubusercontent.com/654645/84791488-abda1800-afc0-11ea-8b20-80e2f7a1bd24.png)|
|![image](https://user-images.githubusercontent.com/654645/84791388-84834b00-afc0-11ea-867d-82412d822242.png)|![image](https://user-images.githubusercontent.com/654645/84791409-8c42ef80-afc0-11ea-99ba-1ae788e7e703.png)|

[Page with multiple links](https://www.consumerfinance.gov/consumer-tools/credit-reports-and-scores/consumer-reporting-companies/), on desktop and mobile:

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/654645/84791818-1e4af800-afc1-11ea-82e4-724e113ea151.png)|![image](https://user-images.githubusercontent.com/654645/84791839-273bc980-afc1-11ea-9f02-9317eea57e19.png)|
|![image](https://user-images.githubusercontent.com/654645/84791855-2dca4100-afc1-11ea-9d2a-99d3ea59f017.png)|![image](https://user-images.githubusercontent.com/654645/84791876-36bb1280-afc1-11ea-869e-03c9134a19eb.png)|

(I fixed the markup to remove that highlighted space before the download icon as well.)

Ping @huetingj for your thoughts on making this change.

Reply:
We can keep it without the link class, but should document that pattern in DS, IMO.

Reply:
Per team discussion, I've added this change to this PR.

Reply:
Ah, good, was just about to comment on this. Yes, the After here is definitely the correct pattern.